source/solvers/cahn_hilliard.cc

```diff
@@ -399,11 +405,11 @@ CahnHilliard<dim>::calculate_phase_statistics()
   if (simulation_parameters.post_processing.verbosity ==
       Parameters::Verbosity::verbose)
     {
-      this->pcout << "Phase statistics : " << std::endl;
-      this->pcout << "\t     Min : " << min_phase_value << std::endl;
-      this->pcout << "\t     Max : " << max_phase_value << std::endl;
-      this->pcout << "\t Average : " << phase_average << std::endl;
-      this->pcout << "\t Integral : " << integral << std::endl;
+      this->pcout << "Phase statistics: " << std::endl;
```

Comment:
You could use the announce string with "-" as separator here to look like the VANS results.

Reply:
The results could be left aligned as well.